.gitattributes

```
@@ -0,0 +1,3 @@
+/src/init/starship.* text eol=lf
+/src/init/mod.rs text eol=lf
```

Init is basically mod plus a bunch of scripts so I guess we can fold it back to:

```suggestion
/src/init/ text eol=lf
```

Obviously, this is nitpicking do as you think it's the best.